### ISSUE TITLE
Merge failures are not obvious

### ISSUE THREAD

**Issue by lakruzz** [Praqma/pretested-integration-plugin#1]
[JENKINS-31135](https://issues.jenkins-ci.org/browse/JENKINS-31135)
---

When a merge fails, it is not obvious why the build is red.
- [ ] It doesn't show in the build description (like other problems)

That could be the first simple solutions.

We should look into other solutions as well: 

- [ ] Show console snippets on the job description.
- [ ] Build bade icons

**Comment by lakruzz** [comment]
@MadsNielsen:
>One simple solution...we have the Nothing to do message like so `build.setDescription(String.format("Noting to do"));`. It should be fairly simple just to add this for merge failures.

**Comment by lakruzz** [comment]
@buep:
>Agree, if you mean that we will set the build description to "Merge failure" when there is a merge failure (and not nothing to do).

**Comment by buep** [comment]
Yes, exactly as we did in earlier releases as far as I remember.
